File: Python/countApplesAndOranges.cs
public static void countApplesAndOranges(int s, int t, int a, int b, List<int> apples, List<int> oranges)
{

        int insideApple = 0;
        int insideOrange = 0;
        foreach(int apple in apples){
            if(apple + a >= s && apple + a <=t){
                insideApple = insideApple+1;
            }
        }
        
        foreach(int orange in oranges){
            if(orange + b >= s && orange + b <=t){
                insideOrange = insideOrange+1;
            }
        }
        
        Console.WriteLine(insideApple);
        Console.WriteLine(insideOrange);
}

File: Python/countApplesAndOranges.py
def countApplesAndOranges(s, t, a, b, apples, oranges):
    # Write your code here
    applesUpdated = [apple + a for apple in apples] 
    orangesUpdated = [orange + b for orange in oranges]
    insideAppleCount = 0
    insideOrangeCount = 0
    for apple in applesUpdated:
        if apple >= s and apple <= t:
            insideAppleCount = insideAppleCount + 1
            
    for orange in orangesUpdated:
        if orange >= s and orange <= t:
            insideOrangeCount = insideOrangeCount + 1
    print(insideAppleCount)
    print(insideOrangeCount)